src/containers/ProfilePage/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Container, Row, Accordion } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import UserDetailForm from "../../components/profile/UserDetailForm";
+import UserEmailForm from "../../components/profile/UserEmailForm";
+import UserPasswordForm from "../../components/profile/UserPasswordForm";
+import defaultProfileImage from "../../images/defaultProfileImage.png";
+import ProfileSection from "./ProfileSection";
+import "./index.scss";
+
+export default function ProfilePage({ submit }) {
+  const { register, handleSubmit, errors } = useForm();
+
+  const SECTIONS = [
```

Comment:
if its a constant thats all caps it doesnt go inside a component. the capitalization indicates that its a constant that doesnt depend on anything in the scope. so this should be `const sections`